File: front/src/models/imc.ts
import { Aluno } from "./aluno";

export interface Imc {
  imcId?: string;
  altura: string;
  peso: string;
  classificacao: number;
  faixa: string;
  alunoId: number;
  criadoEm?: string;
  aluno?: Aluno;
}

File: API/API/Models/Imc.cs
namespace API.Models;

public class Imc
{
    public int Id { get; set; }
    public double Altura { get; set; }
    public double Peso { get; set; }
    public string? Classificacao { get; set; }
    public double Faixa { get; set; }
    public int AlunoId { get; set; }
    public DateTime CriadoEm { get; set; } = DateTime.Now;
    public Aluno? Aluno { get; set; }

}
